modules/web/js/ballerina/diagram2/views/default/components/nodes/if-node.jsx

```diff
@@ -17,12 +17,15 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
+import _ from 'lodash';
 import CompoundStatementDecorator from './compound-statement-decorator';
 import TreeUtil from './../../../../../model/tree-util';
 import IfNodeModel from './../../../../../model/tree/if-node';
 import DropZone from './../../../../../drag-drop/DropZone';
 import DefaultNodeFactory from './../../../../../model/default-node-factory';
 import AddCompoundBlock from './add-compound-block';
+import FragmentUtils from './../../../../../utils/fragment-utils';
+import TreeBuilder from './../../../../../model/tree-builder';
```

Comment:
use absolute paths to import modules. https://github.com/ballerinalang/composer/wiki/Developer-best-practices#use-absolute-paths-to-import-modules

Reply:
Fixed with f580f482cc82e92dd391ea225593c001ed2f5183